modules/nf-core/svanalyzer/svbenchmark/main.nf

```diff
@@ -0,0 +1,68 @@
+
+process SVANALYZER_SVBENCHMARK {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/svanalyzer:0.35--pl526_0':
+        'biocontainers/svanalyzer:0.35--pl526_0' }"
+
+    input:
+    tuple val(meta), path(test)
+    tuple val(meta2), path(truth)
+    path(fasta)
+    path(fai)
+    path(bed)
+
+    output:
+    tuple val(meta), path("*.falsenegatives.vcf"), emit: fns
+    tuple val(meta), path("*.falsepositives.vcf"), emit: fps
```

Comment:
Is `bgzip` present in the container? If so, I would suggest compressing these as that's the nf-core standard

Reply:
yeap here. will do it.